woocommerce/woocommerce
# Drop Select2 in Backwards compatable manner

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #48577 

Given that Select2 and SelectWoo have the same APIs, this PR points the registry of the Select2 library to SelectWoo. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Please test it with the following instructions:

- Open `\wp-admin > advanced > Webhooks` page
- In `wc-enchanced-select.js`, replace `.selectWoo` with `.select2` everywhere in the file.
- Refresh and clear browser cache if required.
- Verify if the file is loaded with `select2` by going to `browser console > sources > cmd + p > search for wc-enhanced-select.js > verify if it has select2`.
- Select Dropdown should work as expected in `\wp-admin > advanced > Webhooks` page

![image](https://github.com/woocommerce/woocommerce/assets/15019298/cd2d22c8-0afb-4741-afd7-bf1e128583a3)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
